liota/dccs/iotcc.py

```diff
@@ -332,16 +331,33 @@ def _create_iotcc_json(self):
         return iotcc_path
 
     def store_reg_entity_details(self, entity_type, entity_name, reg_entity_id, entity_local_uuid):
-        msg = ''
         if self._iotcc_json == '':
             log.warn('iotcc.json file missing')
             return
         try:
-            with open(self._iotcc_json, 'r') as f:
-                msg = json.load(f)
-            f.close()
+            f = open(self._iotcc_json, 'r')
```

Comment:
I believe you were going to add the lock mechanism over here.

Reply:
No, I uses a lock before calling this func rather than inside it. 
        if entity_obj.entity_type == "HelixGateway":
 -                self.store_reg_entity_details(entity_obj.entity_type, entity_obj.name, self.reg_entity_id,
 -                                              entity_obj.entity_id)
                  with self.file_ops_lock:
 +                    self.store_reg_entity_details(entity_obj.entity_type, entity_obj.name, self.reg_entity_id,
 +                                              entity_obj.entity_id)
                      self.store_reg_entity_attributes("EdgeSystem", entity_obj.name,
                                                       self.reg_entity_id, None, None)

Reply:
Okay